scripts/quay.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+
+app="${1}"
+version="${2}"
+
+echo "Logging into Quay..."
+echo "${QUAY_PASSWORD}" | docker login -u "${QUAY_USERNAME}" --password-stdin quay.io
```

Comment:
where are the QUAY_PASSWORD and QUAY_USERNAME coming from? Only Docker username and password are mentioned in the docker-publish.yml